src/Resources/public/administration/js/fin-search.js

```diff
@@ -1 +1 @@
1,this.isLoading=!1})},_setErrorStates(e=!1){this.isLoading=!1,this.shopkeyAvailable?this.isValidShopkey?!1===this.isRegisteredShopkey?this.shopkeyErrorState={code:1,detail:this.$tc("findologic.notRegisteredShopkey")}:this.shopkeyErrorState=null:this.shopkeyErrorState={code:1,detail:this.$tc("findologic.invalidShopkey")}:this.shopkeyErrorState={code:1,detail:this.$tc("findologic.fieldRequired")},e&&this._showNotification()},_showNotification(){this.shopkeyAvailable?this.isValidShopkey?!1===this.isRegisteredShopkey&&this.createNotificationError({title:this.$tc("findologic.settingForm.titleError"),message:this.$tc("findologic.notRegisteredShopkey")}):this.createNotificationError({title:this.$tc("findologic.settingForm.titleError"),message:this.$tc("findologic.invalidShopkey")}):this.createNotificationError({title:this.$tc("findologic.settingForm.titleError"),message:this.$tc("findologic.fieldRequired")})},_validateShopkeyFromService(){return this.isLoading=!0,this.httpClient.get(`https://account.findologic.com/api/v1/shopkey/validate/${this._getShopkey()}`).then(e=>String(e.status).startsWith("2")).catch(()=>!1)}}});var g=t("iH4O"),h=t.n(g);const{Component:p,Mixin:f}=Shopware,{Criteria:u}=Shopware.Data;p.register("findologic-config",{template:h.a,name:"FindologicConfig",inject:["repositoryFactory"],mixins:[f.getByName("notification")],props:{actualConfigData:{type:Object,required:!0},allConfigs:{type:Object,required:!0},shopkeyErrorState:{required:!0},selectedSalesChannelId:{type:String,required:!1,default:null},isStagingShop:{type:Boolean,required:!0,default:!1},isValidShopkey:{type:Boolean,required:!0,default:!1},isActive:{type:Boolean,required:!0,default:!1},shopkeyAvailable:{type:Boolean,required:!0,default:!1}},data:()=>({isLoading:!1}),methods:{checkTextFieldInheritance:e=>"string"!=typeof e||e.length<=0,checkBoolFieldInheritance:e=>"boolean"!=typeof e,openSalesChannelUrl(){if(null!==this.selectedSalesChannelId){const e=new u;e.addFilter(u.equals("id",this.selectedSalesChannelId)),e.setLimit(1),e.addAssociation("domains"),this.salesChannelRepository.search(e,Shopware.Context.api).then(e=>{const n=e.first().domains.first();this._openStagingUrl(n)})}else this._openDefaultUrl()},_openDefaultUrl(){const e=`${window.location.origin}?findologic=on`;window.open(e,"_blank")},_openStagingUrl(e){if(e){const n=`${e.url}?findologic=on`;window.open(n,"_blank")}else this._openDefaultUrl()}},computed:{showTestButton(){return this.isActive&&this.shopkeyAvailable&&this.isValidShopkey&&this.isStagingShop},salesChannelRepository(){return this.repositoryFactory.create("sales_channel")}}});var S=t("kfPE"),k=t("KBz0");const{Module:m}=Shopware;m.register("findologic-module",{type:"plugin",name:"FinSearch",title:"findologic.header",description:"findologic.general.mainMenuDescription",color:"#f7ff0f",snippets:{"de-DE":k,"en-GB":S},routes:{index:{component:"findologic-page",path:"index",meta:{parentPath:"sw.settings.index"}}}})}},[["xZag","runtime","vendors-node"]]]);
```

Comment:
Why was this built, but there were no changes in the JS?

Reply:
Not sure. Might be due to some changes in 6.3.